tensorboard/components/tf_runs_selector/tf-runs-selector.html

```diff
@@ -55,7 +58,10 @@ <h3 id="tooltip-help" class="tooltip-container">Runs</h3>
     </paper-button>
     <template is="dom-if" if="[[dataLocation]]">
       <div id="data-location">
-        <tf-wbr-string value="[[_clippedDataLocation]]" /><!--
+        <tf-wbr-string
+          value="[[_clippedDataLocation]]"
+          delimiter-pattern="[[_dataLocationDelimiterPattern]]"
```

Comment:
You can just inline the literal string here, which removes the need for
the extra (read-only, constant) property.

Reply:
I use it twice in the file, though. When I was playing around with the tf-wbr-string API today it was good to have one less place to change it.